### ISSUE TITLE
mimemagic-0.3.5 issue still not fixed

### ISSUE THREAD

**Issue by acoolstraw** [mastodon/mastodon#16102]
### Expected behaviour
I followed Mastodon's official tutorial, and it should have let me do `bundle install`

### Actual behaviour
It couldn't continue because of this error:
```
~/live$ bundle install -j$(getconf _NPROCESSORS_ONLN)
Fetching gem metadata from https://rubygems.org/............
Your bundle is locked to mimemagic (0.3.5) from rubygems repository
https://rubygems.org/ or installed locally, but that version can no longer be
found in that source. That means the author of mimemagic (0.3.5) has removed it.
You'll need to update your bundle to a version other than mimemagic (0.3.5) that
hasn't been removed in order to install.
```

### Steps to reproduce the problem
Just follow the official tutorial

### Specifications
I found it on the `live` version

**Comment by ClearlyClaire** [comment]
It's fixed in the development version, there's just no release with it yet, unfortunately.

**Comment by RobPurcellUK** [comment]
This seems like a thing which should have received a release (3.3.1?). It blocks people from installing/upgrading a server completely without amending code and config themselves.

**Comment by alx** [comment]
Hi,

Doing a fresh install and got this issue too.

https://docs.joinmastodon.org/admin/install/ 

Could the documentation be modified ? Fixed using `main` git branch:

```
git checkout main
bundle install -j$(getconf _NPROCESSORS_ONLN)
...
```

**Comment by wilkis3** [comment]
Issue is solved in 3.4.0-rc2 or ``main`` branch.
Manual to fix this under https://github.com/tootsuite/mastodon/issues/16228#issuecomment-839731734
